### ISSUE TITLE
The dataloader of SensatUrban with point-transformer?

### ISSUE THREAD

**Issue by whuhxb** [meidachen/STPLS3D#6]
Hi @meidachen 

How to set the dataloader of SensatUrban dataset with point-transformer? As SensatUrban and STPLS3D datasets are both large-scale, could I refer to and follow the generate_blocks.py and stpls.py upon SensatUrban dataset with point-transformer code?

Thanks.

**Comment by RockyatASU** [comment]
Thanks for your interest in our datasets. Yes, you can follow the generate_blocks.py and stpls.py to create a new dataset class for the SensateUrban dataset. As the point cloud resolution of SensatUrban and STPLS3D datasets are different, you may need to change the default block size in generate_blocks.py.

**Comment by whuhxb** [comment]
@RockyatASU 

OK  Thanks a lot

**Comment by RockyatASU** [comment]
Hi @whuhxb,

I think our STPLS3D has a similar point cloud resolution to the SensatUrban dataset so we recommend you try 0.3~0.5 voxel size for your experiments on SensatUrban. For details please refer to the author of SensatUrban. 

You can refer to this [link](https://github.com/POSTECH-CVLab/point-transformer/issues/32) for the detailed parameters of point-transformer. Basically, point-transformer utilizes the voxelized point cloud to reduce the point cloud density during training but it still produces a semantic label for each point. Sure, you can feed one whole scene of SensatUrban into a point-transformer without generating subblocks and the setting of RandLA-Net may be a good starting point for your experiments.

**Comment by whuhxb** [comment]
Hi @RockyatASU，

Thank you for your kind and detailed reply.  I will have a try. 
In addition, for S3DIS dataset with area 5 as testing, Point Transformer uses 12346 areas as training and 5 area as evaluate to obtain the best model and then uses 5 as testing, which I think is not reasonable as train val and test set are no intersection. But I have seen several github codes on S3DIS dataset adopts this training setting. I have no idea why set like this.
I wonder to know whether Point Transformer for STPLS3D also adopts the same setting. WMSC is for evaluate and testing and other files are used for training. 

Thanks.
